### ISSUE TITLE
Online Students in Class

### ISSUE THREAD

**Issue by JoeJimFlood** [uwhpsc-2016/syllabus#39]
I'm an EDGE student currently living in Indianapolis, but I'll be in town on Thursday for the MS Symposium. Would there be any problems if I showed up to class that day?

**Comment by cswiercz** [comment]
Not at all! The room is supposed to hold about 40 and only about 8 are showing up so there is plenty of room for online students.

Please introduce yourself! I like matching faces to screen names.

**Comment by cswiercz** [comment]
# Open Invitiation

Any online students who would like to attend lectures in class is welcome to. There is plenty of space to go around.
